src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int MAX_RANDOM_PROPERTIES = 50;
+
+    public Figure getRandomFigure() {
+        int randomFigureValue = new Random().nextInt(NameOfFigure.values().length);
+        String figureName = NameOfFigure.values()[randomFigureValue].toString();
+        ColorSupplier colorSupplier = new ColorSupplier();
+
+        switch (figureName) {
+            case "Circle":
```

Comment:
better switch by values
```suggestion

        switch (randomFigureValue) {
            case 0:
```
and so on